stories/core/Tile.stories.tsx

```diff
@@ -131,6 +133,68 @@ AllProps.argTypes = { ...Basic.argTypes };
 AllProps.args = {
   ...Basic.args,
   isSelected: true,
+  isActionable: true,
```

Comment:
Why passing like that? It is not visible to the users how props look.

Reply:
What do you mean its not? I also destructure in the code itself and assign default value